airbyte-integrations/connectors/source-marketo/source_marketo/utils.py

```diff
@@ -0,0 +1,92 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+
+from datetime import datetime
+
+STRING_TYPES = [
+    "string",
+    "email",
+    "reference",
+    "url",
+    "phone",
+    "textarea",
+    "text",
+    "lead_function",
+]
+
+
+def to_datetime_str(date: datetime) -> str:
+    """
+    Returns the formated datetime string.
+    :: Output example: '2021-07-15T0:0:0Z' FORMAT : "%Y-%m-%dT%H:%M:%SZ"
+    """
+    return date.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
+def clean_string(string: str) -> str:
+    """
+    input -> output
+    "updatedAt" -> "updated_at"
+    "UpdatedAt" -> "updated_at"
+    "base URL" -> "base_url"
+    "UPdatedAt" -> "u_pdated_at"
+    "updated_at" -> "updated_at"
+    " updated_at " -> "updated_at"
+    "updatedat" -> "updatedat"
```

Comment:
is there a marketo docs link summarising these conversions we can link here?

Reply:
yes.